create_cluster.py

```diff
@@ -0,0 +1,99 @@
+import numpy as np
+import dataclasses
+import matplotlib.pyplot as plt
+import seaborn as sns
+import scipy.stats as stats
+import pandas as pd
+import warnings
+import joblib
+import sklearn
+from typing import List
+from sklearn.cluster import KMeans
+from sklearn.datasets import make_blobs
+import json
+from sklearn.utils._openmp_helpers import _openmp_effective_n_threads
+import os
+import datetime
+
+from google.colab import auth
+import gspread
+from google.auth import default  #authenticatiing to google
+from clusters.py import all
```

Comment:
I'm not convinced that moving the cluster routine to a new function is a good idea.

Most importantly, imports are weird in Python, and depending on how the package is set up, this might or might not be found.

More importantly, when you do an import all, it's not clear later where a string comes from.  Thus it is not allowed by the Google style.  Now I understand why.